accounting/fiscal_localizations/overview/fiscal_localization_packages.rst

```diff
@@ -0,0 +1,47 @@
+============================
+Fiscal Localization Packages
+============================
+
+**Fiscal Localization Packages** are country-specific modules that install pre-configured taxes,
+fiscal positions, chart of accounts, and legal statements on your database. Some additional features,
+such as the configuration of specific certificates, are also added to your Accounting app, following
+your fiscal administration requirements.
+
+.. note::
+   Odoo continuously adds new localizations and improves the existing packages.
+
+Configuration
+=============
+
+Odoo automatically installs the appropriate package for your company, according to the country
+selected at the creation of the database.
```

Comment:
```suggestion
Odoo automatically installs the appropriate package(s) for your company, according to the country
selected at the creation of the database.
```

Reply:
Just a detail but, based on the following paragraph, I think it can be useful to already open the possibility to have several packages.

Reply:
In this case, Odoo only installs one package when you create a new database, so I'll leave it singular. But well spotted! I appreciate the attention to such details :D respect++

Reply:
Hmm, ok, I didn't know.
But you confirm that you can install new packages later? No limit in the number of packages?